build_tools/github_actions/runner/gcp/image_setup.sh

```diff
@@ -188,13 +188,25 @@ EOF
 
   local gcloud_checksum="e0382917353272655959bb650643c5df72c85de326a720b97e562bb6ea4478b1"
 
-  nice_curl \
-    https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-414.0.0-linux-x86_64.tar.gz \
-    --output gcloud.tar.gz
-  echo "${gcloud_checksum} *gcloud.tar.gz" | sha256sum --check --strict
-  tar -xf gcloud.tar.gz
-  rm gcloud.tar.gz
-  google-cloud-sdk/install.sh --quiet
+  GCLOUD_CLI_VERSION="414.0.0"
+  # Gcloud CLI versioned archive not available for Arm64 Linux
```

Comment:
I think https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-414.0.0-linux-arm.tar.gz might work. But feel free to keep this current solution.

Reply:
Thx! I didn't see that one. I checked here: https://cloud.google.com/sdk/docs/downloads-versioned-archives, but it's not listed (I sent that feedback to the site...). I didn't find it via a Google search either.

Anyways, will fix.